### PR TITLE
Remove duplicate warnings for unhandled summary keywords.

### PR DESCRIPTION
**Pull request by atgeirr**
In the current master, Norne generates more than 1100 warnings for unhandled summary keywords. Many of those are duplicates, because the same warning will be repeated for all involved wells. With this PR, we will only see each (keyword, location) combination warned about once. This reduces the number of such warnings on Norne to 87 or so.

This restores older warning behaviour in terms of numbers of warnings, but keeps the most important feature that was added: printing the deck location of the triggering keyword. Furthermore, the warnings are now sorted by location first rather than by keyword first.

### CODE REVIEW

**Comment by atgeirr** [comment]
jenkins build this please

**Comment by atgeirr** [comment]
@joakim-hove can you review this? The change that caused the excessive warnings was in #1204.

File: src/opm/output/eclipse/Summary.cpp [hunk 1]
```diff
@@ -1818,8 +1818,26 @@ namespace Evaluator {
 
 void reportUnsupportedKeywords(std::vector<Opm::SummaryNode> keywords)
 {
-    std::sort(keywords.begin(), keywords.end());
-    auto uend = std::unique(keywords.begin(), keywords.end());
+    // Sort by location first, then keyword.
+    auto loc_kw_ordering = [](const Opm::SummaryNode& n1, const Opm::SummaryNode& n2) {
+        if (n1.location() == n2.location()) {
```
Comment (joakim-hove) [inline src/opm/output/eclipse/Summary.cpp]: I would have written this with the `else` blocks:
```C++
if (n1.location() == n2.location())
     return ...

if ( )
     return 

return;
```

Comment (atgeirr) [reply src/opm/output/eclipse/Summary.cpp]: Amended and force pushed (assuming you meant *without*, not *with*).

**Comment by atgeirr** [comment]
jenkins build this please

File: src/opm/output/eclipse/Summary.cpp (hunk 1 above)
Comment (joakim-hove) [reply src/opm/output/eclipse/Summary.cpp]: Ahhh dang - you kept the braces; naught boy!

Comment (atgeirr) [reply src/opm/output/eclipse/Summary.cpp]: Sorry, it did not really cross my mind! Not intending to be naughty.

Comment (joakim-hove) [reply src/opm/output/eclipse/Summary.cpp]: :smile:

**Pull request merged**